contracts/examples/NonMerklizedIdentityExample.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.16;
+
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { IState } from "@iden3/contracts/interfaces/IState.sol";
+import { ClaimBuilder } from "@iden3/contracts/lib/ClaimBuilder.sol";
+import { IdentityLib } from "@iden3/contracts/lib/IdentityLib.sol";
+import { IdentityBase } from "@iden3/contracts/lib/IdentityBase.sol";
+import { PrimitiveTypeUtils } from "@iden3/contracts/lib/PrimitiveTypeUtils.sol";
+
+/**
+ * @dev Contract managing nonmerklized onchain identity
+ */
+contract NonMerklizedIdentityExample is IdentityBase, OwnableUpgradeable {
+    using IdentityLib for IdentityLib.Data;
+
+    // ClaimInfo represents a claim and its metadata
+    struct ClaimInfo {
+        // metadata
+        string jsonldSchemaURL;
+        uint256 jsonldSchemaHash;
+        string jsonSchemaURL;
+        string credentialType;
+        // data
+        uint256[8] claim;
+    }
+
+    uint256[500] private __gap_before;
+
+    // These variables are used to create core claims
+    // They represent the type of claim
+    string private jsonldSchemaURL;
+    uint256 private jsonldSchemaHash;
+    string private jsonSchemaURL;
+    string private credentialType;
+
+    // countOfIssuedClaims count of issued claims for incrementing revocation nonce for new claims
+    uint64 private countOfIssuedClaims = 0;
+   
+    // claimsMap claims storage
+    mapping(uint256 => mapping(string => ClaimInfo)) private claimsMap;
+   
+    uint256[44] private __gap_after;
+
+    function initialize(
+        address _stateContractAddr,
+        string calldata _jsonldSchemaURL, 
+        uint256 _jsonldSchemaHash,
+        string calldata _jsonSchemaURL,
+        string calldata _credentialType
+    ) public initializer {
+        jsonldSchemaURL = _jsonldSchemaURL;
+        jsonldSchemaHash = _jsonldSchemaHash;
+        jsonSchemaURL = _jsonSchemaURL;
+        credentialType = _credentialType;
+
+        IdentityBase.initialize(_stateContractAddr);
+        __Ownable_init();
+    }
+
+    // saveClaim save a claim to storage
+    function saveClaim(
+        uint256 _id,
+        string memory _uuid,
```

Comment:
use calldata for _uuid

Reply:
How do you propose to get a specific claim?